cmake/external_llvm_prebuilt.cmake

```diff
@@ -143,10 +143,12 @@ set(LLVM_LINK_LIBS
     ${INSTALL_DIR}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}LLVMDemangle${CMAKE_STATIC_LIBRARY_SUFFIX}
 )
 
-if(APPLE)
-    set(LLVM_LINK_LIBS ${LLVM_LINK_LIBS} curses z m)
-else()
-    set(LLVM_LINK_LIBS ${LLVM_LINK_LIBS} tinfo z m)
+if(UNIX OR APPLE)
+    if(APPLE)
```

Comment:
One case for UNIX, one for APPLE, no need to nest in a UNIX or APPLE.

Reply:
Done.